assets/section-image-banner.css

```diff
@@ -213,6 +198,7 @@
     --color-button: 255, 255, 255;
     --color-button-text: 0, 0, 0;
     max-width: 89rem;
+    padding: 0;
```

Comment:
I believe in a previous PR we actually increased the width of the box from what I recall - which is why this can go wider now🤔 

cc @melissaperreault @wiktoriaswiecicka

Reply:
I believe we could adjust when text is aligned left or right. It doesn't require the same `max-width` as if it was centered. Is this possible?

Reply:
Yes, we could make the max-width smaller by adding conditional styles. What were you thinking in terms of the width in those cases @melissaperreault ?

Reply:
It looks like Wik suggested 68rem let me try that

Reply:
As per Wik's suggestion updated it to be 68rem when:

- left / right aligned? OR when the heading is set to medium

https://github.com/Shopify/dawn/pull/900/commits/c11cb02ec2e2ac2a59ff3767791533b8e912dad6

cc @wiktoriaswiecicka @melissaperreault @ludoboludo

Reply:
To confirm - We're keeping 68rem for left & right alignment 👍